mail/dovecot2/Portfile

```diff
@@ -80,6 +79,33 @@ configure.args      --sysconfdir=${prefix}/etc \
 
 configure.cppflags  -I${prefix}/include/openssl
 
+subport dovecot2-sieve {
+
+    github.setup        dovecot pigeonhole 0.5.7.2
+    homepage            https://pigeonhole.dovecot.org/
+    description         Pigeonhole sieve and managesieve plugins for dovecot
+    long_description    ${description}
+    distname            dovecot-${core-version}-pigeonhole-${version}
+    checksums           rmd160  75e8312c59bce686ef191d7a939c6dccd343024b \
+                        sha256  23aff6a6ffee6d4b970a24862161bfbae07b0e16cf7446854c728bfccfe34929 \
+                        size    1026859
+
+    depends_lib         port:mercurial
```

Comment:
It is best practice to use `depends_lib-append` when using portgroups so you don't overwrite any current or future portgroup dependencies.